client/src/components/PhotoGallery.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import PhotoGalleryItem from './PhotoGalleryItem';
+import '../css-components/PhotoGallery.css';
+
+const PhotoGallery = ({ photos }) => (
+  <div className="gallery-div">
+    <img className="main-image" src={photos[2]} alt="random pic for house listing" />
+    {photos.map((photo) => (
+      <PhotoGalleryItem key={photo} photo={photo} />
```

Comment:
each photo is random right, so is there a possibility that one key might equal another? I would suggest adding in an index param next to photo on line 10, and then combining the photo and index to be the key